integration_test/third_party_apps_data/applications/aerospike/debian_ubuntu/install

```diff
@@ -17,20 +37,21 @@ case $ID in
         case "$VERSION_ID" in
             18*) ubuntu_version="ubuntu18.04";;
             20*) ubuntu_version="ubuntu20.04";;
-            *) echo "Found version: $VERSION_ID, supported versions are 18/20"
+            23*) ubuntu_version="ubuntu22.04";;
+            *) echo "Found version: $VERSION_ID, supported versions are 18/20/23"
         esac
         os_version=$ubuntu_version
         ;;
 esac
 
-# download, extract, install, and start aerospike community edition
-wget -O aerospike.tgz "https://download.aerospike.com/artifacts/aerospike-server-community/5.7.0.23/aerospike-server-community-5.7.0.23-$os_version.tgz"
-tar -xvf aerospike.tgz
-(cd aerospike-server-community-5.7.0.23-*/ && sudo ./asinstall)
+case $os_version in
+    debian9|ubuntu18.04)
+        install_previous;;
+    debian10|debian11|ubuntu20.04|ubuntu22.04)
```

Comment:
Does this not change the behavior on Debian 10/11 and Ubuntu 20.04? Any reason for that change?

Reply:
Correct me if I'm wrong @dehaansa, but I believe the reasoning here was that we should be using the most current version of Aerospike for all OS versions that support it, and specific previous versions of Aerospike should only be used when absolutely necessary (as is the case for Debian 9 and Ubuntu 18.04).

Reply:
As we were required to use the more recent release of aerospike for lunar support, I opted to upgrade to use the same version where possible and leave the 5.7.x release only for distributions that didn't have a package available for the  6.3.x release line. If a more minimal change would be better we can certainly only add the 6.3.x line for Lunar.

Reply:
Nah, upgrading is fine, just flagging it to make sure it was intended. Thanks.